kubernetes/start

```diff
@@ -63,7 +63,7 @@ done
 
 if $(which kubelet >&/dev/null); then
   log "Running setup-environment script"
-  bash setup-environment &
+  ./setup-environment
```

Comment:
If you don't run this in the background, the script will stop here until it returns. But it will never do, because setup-environment script needs the kubelet to be started which only happens on line 73 below.